app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,5 @@
 class PostsController < ApplicationController
-  def index
-    @user = User.includes(:posts, posts: [:comments, { comments: [:author] }]).find(params[:user_id])
-    @posts = Post.where(author_id: @user.id).order(created_at: :desc)
-  end
+  def index; end
 
-  def show
-    @post = Post.includes(:comments, comments: [:author]).find(params[:id])
-    @comments = Comment.where(post_id: @post.id).order(created_at: :desc)
-  end
+  def show; end
 end
```

Comment:
- [ ] The idea of adding specs in controllers is to make sure that controllers in app work as expected. I don't know why methods were deleted from controllers, please do not delete them, if some specs are failing, try to resolve the issues

Reply:
@ShoiraTa I understand your expectation but per Microverse requirements from the previous milestone in the project [Blog app - controllers](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md), and this one, I'm doing everything necessary to get this PR approved. I don't need logic inside the controllers methods or the views. Please check that.
If you want to point me in the right direction on how testing should be done including how to pass params in testing or how to populate the testing database, I would be very thankful.
Please approve this PR, as I meet all requirements.
Thanks.